category_maps.py

```diff
@@ -0,0 +1,98 @@
+### count categories
+## data_type count (for sample type, column B)
+data_type_map = {
+    'brain_types' : ['brain hemisphere', 
+                     'brain region', 
+                     'brain section set', 
+                     'brain slice', 
+                     'whole brain', 
+                     'spinal cord'],
+    'cell_types' : ['cell body', 
```

Comment:
Similarly, `cell_types` has a specific usage that does not align with this. It would be more clear to say `cell_sample_types` or some such.